src/index.js

```diff
@@ -68,6 +68,23 @@ function runAsync (command, options) {
   })
 }
 
+function runAsyncSpawn (command, args, options) {
+  return new Promise((resolve, reject) => {
+    const spawnOptions = Object.assign({shell: true}, options)
+    const asyncProcess = childProcess.spawn(command, args, spawnOptions)
```

Comment:
I've tested `spawn` with `shell: true` options and it seems that we can call the command with args as a one string. Looks like there is no need to have args separately then.